NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -299,6 +318,107 @@ public RichTextBoxTargetMessageRetentionStrategy MessageRetention
         private volatile Queue<MessageInfo> messageQueue = null;
 
         /// <summary>
+        /// If set to true, using "rtb-link" renderer (<see cref="RichTextBoxLinkLayoutRenderer"/>) would create clickable links in the control.
+        /// <seealso cref="LinkClicked"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool SupportLinks 
+        {
+            get { return supportLinks; }
+            set
+            {
+                supportLinks = value;
+                if (supportLinks)
+                {
+                    lock (linkRegexLock)
+                    {
+                        if (linkAddRegex == null)
```

Comment:
This is a bit tricky IMO. I think this lazy init should be encapsulated in a property. Now we can use `linkAddRegex` and `linkRemoveRtfRegex` too soon (theoretical), which lead to unexpected behavior / nullref ex.

When encapsulating in a property this is also possible, but we can find easily all usages of the field (and can be replaced with the property).

Do you agree?